includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -421,4 +411,91 @@ private function getPartnerSlot() {
 		}
 		return null;
 	}
+
+	private function getLogo() {
+		$logo = [
+			// Deprecated
+			'header' => [
+				'type' => 'link-image',
+				'href' => $this->getHref( 'fandom-logo' ),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "fandom-logo" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)